velox/exec/Operator.cpp

```diff
@@ -323,8 +323,11 @@ void OperatorStats::add(const OperatorStats& other) {
 
   memoryStats.add(other.memoryStats);
 
-  for (const auto& stat : other.runtimeStats) {
-    runtimeStats[stat.first].merge(stat.second);
+  for (const auto& [name, stats] : other.runtimeStats) {
+    if (UNLIKELY(runtimeStats.count(name) == 0)) {
+      runtimeStats.insert(std::make_pair(name, RuntimeMetric(stats.unit)));
```

Comment:
Do you need to create new RuntimeMetric instance? Can you instead use 'stats' ? then move the code on 330 into the 'else' clause?